From e9ace785c6250783454e838f1183cc353d5837d0 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 23 Oct 2020 05:59:53 -0700
Subject: [PATCH 1726/1851] dt-bindings: display: xlnx: sdi: Update for PICXO

commit a921e3cbe15df55390327028309f4ce5eecbc104 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds 2 boolean optional properties for controlling qpll,
picxo reset and programming.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt         | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
index a6f8640a3a2c..6edc2236c28d 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
@@ -24,7 +24,9 @@ Optional properties:
  - xlnx,tx-insert-c-str-st352: Insert ST352 payload in Chroma stream.
  - interrupt-names: Should be "sdi_tx_irq". This is only required when
    multiple interrupts are connected in the hardware design.
- - picxo-reset-gpio: Specifier for a gpio that asserts picxo core.
+ - xlnx,qpll1enabled: boolean property present when the QPLL1 drives GT clock
+   absent with QPLL0 drives GT clock
+- xlnx,picxo_enabled: boolean property present when PICXO is present in design
 
 Example:
 
@@ -40,6 +42,7 @@ Example:
 		clock-names = "s_axi_aclk", "video_in_clk", "sdi_tx_clk";
 		clocks = <&misc_clk_0>, <&misc_clk_1>, <&misc_clk_2>;
 		phy-reset-gpio = <&axi_gpio_0 0 0 0>;
+		xlnx,qpll1enabled;
 		ports {
 			#address-cells = <1>;
 			#size-cells = <0>;
-- 
2.31.1

